0006-cmx2cdp-protocol/README.md

```diff
@@ -130,28 +138,29 @@ Example:
 
 ### Configuration push
 
-Event-based approach as described in [3/Messaging IPC][3/MIPC] documentation can be used as alternative,
-thus manipulations with endpoint configurations are accepted as events.
+Manipulations with endpoint configurations can be done as events.
```

Comment:
Again: these are no manipulations. This is a broadcast push notification from provider to consumers.